plugins/katello/3.13/release_notes/release_notes.md

```diff
@@ -6,11 +6,62 @@ version: 3.13
 
 # Katello {{ page.version }} Release Notes
 
-For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/master/CHANGELOG.md).
+For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/KATELLO-3.13/CHANGELOG.md).
 
 ## Features
 
-## Deprecation Warnings
+### System Purpose Support
+
+Katello 3.13 provides setting system purpose attributes on Activation Keys in the UI. Additionally, the Hammer CLI suports setting System Purpose attrbutes on Activation Keys and Content Hosts.
```

Comment:
We tend to capitalize System Purpose, let's do the same here

s/suports/supports